packages/react-storage/src/components/StorageBrowser/views/LocationActionView/DeleteView/__tests__/useDeleteView.spec.ts

```diff
@@ -1,7 +1,6 @@
-import { renderHook, act } from '@testing-library/react';
+import { act, renderHook } from '@testing-library/react-hooks';
```

Comment:
Why importing them from the different dep? IIRC we are adviced to use the original one when possible. Same applies to other files